README.md

```diff
@@ -1,13 +1,16 @@
-# isa7-css-ecommerce-project
+# isa7-css-e-commerce-project
+
 ## CSS Assignment - 15 points
-- Instructions (you can design an ecommerce website the you like but make sure it has the following features)
-   - products page (2 points) 
-   - product details page (1 point)
-   - cart page which includes payment details (2 points)
-   - registration page, login page (2 points)
-   - deploy the project on netlify and share the deployment link in README.md; do not forget to make a PR as well (2 points)
-   - apply any relevant animation in the project (1 point)
-   - make sure the website is 100% accessibile (2 point)
-   - make sure the website is 100% responsive (2 points)
-   - Your creativity and additional functionality to the project will help you to get extra 1 point
-  
+
+- Instructions (you can design an e-commerce website that you like but make sure it has the following features)
+  - products page (2 points)
+  - product details page (1 point)
+  - cart page which includes payment details (2 points)
```

Comment:
-You didn't create the details product page yet.
 -You can add a product to the cart with a price and buy button.

Reply:
Also done!
I hope that you would take another look at it.
Later on I still want to apply some JavaScript for the payment calculations.

Thanks for supporting @slmgnm 🌹